docs/LwM2M_Client/Getting_started.md

```diff
@@ -75,3 +75,11 @@ Integrate your nRF9160 board using our Anjay LwM2M Client based on Zephyr OS.
 ![nRF9160](images/nRF9160.png "nRF9160"){ width=70% }
 
 [Start integration](./Nordic/nRF9160.md){: .md-button .md-button--big }
+
+### nRF7002
```

Comment:
```suggestion
### nRF7002 DK
```

Reply:
Well, we should add this suffix everywhere - for nRF9160, Thingy, STM and ESP boards, or nowhere. So I would like to see @malw4 and other people opinion here

Reply:
We can change this in nRF9160 DK. But Thingy...I don't know what you want to add to this

Reply:
Actually - no. There is no `esp32 DK` or `B-L475E-IOT01A2  DK` or either `Thingy:91 DK`, but there is `nRF9160 DK` or `nRF7002 DK`.